### PR TITLE
Fix a bug in batch_eval

### LINKED ISSUE #572: a problem about mnist_blackbox.py
problem

How should i do?

### PR DESCRIPTION
**Pull request by WindQAQ**
Fix #572 .
The problem is caused by the reason that `batch_eval` in `utils_tf.py` does not return any value.

### CODE REVIEW

File: cleverhans/evaluation.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 Includes multi-GPU support for fast evaluation.
 """
 
+import warnings
```
Comment (npapernot) [inline cleverhans/evaluation.py]: Why was this import added?

Comment (WindQAQ) [reply cleverhans/evaluation.py]: Because module `warnings` is used in `batch_eval`, but it has never been imported. See [here](https://github.com/tensorflow/cleverhans/blob/3b4820ab723db3dae549154064d748695e81805d/cleverhans/evaluation.py#L259) for more details.

Comment (npapernot) [reply cleverhans/evaluation.py]: Thanks for catching that! Could you move the import after six.moves so they are in alphabetical order?

Comment (WindQAQ) [reply cleverhans/evaluation.py]: Sure!

**Comment by npapernot** [comment]
I closed and reopened the PR to trigger a new Travis build.

**Comment by WindQAQ** [comment]
@npapernot Should I do anything for the CLA?

**Comment by npapernot** [comment]
No it just broke because I pushed a minor fix to your branch so we can still merge. Thanks for asking!
